system/Files/File.php

```diff
@@ -25,10 +25,8 @@ class File extends SplFileInfo
 {
     /**
      * The files size in bytes
-     *
-     * @var int
      */
-    protected $size;
+    protected int $size;
```

Comment:
This is a breaking change if anyone is extending this class. Please explain or revert.

Reply:
Hi @MGatner , thanks for your reply.
I have now restored the property to its original state.
The reason I will make this change is because of the following.
`
$ignoreErrors[] = [
	'message' => '#^Property CodeIgniter\\\\Files\\\\File\\:\\:\\$size \\(int\\) on left side of \\?\\? is not nullable\\.$#',
	'count' => 1,
	'path' => __DIR__ . '/system/Files/File.php',
];
`